Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM python:buster
+
+RUN apt-get update
+RUN apt-get -y install curl git rsync \
+    # Node version 14
+    && curl -sL https://deb.nodesource.com/setup_14.x | bash \
+    && apt-get install nodejs
+
+
+RUN git clone https://github.com/codetheweb/MTU-Transfer-Course-Gatherer.git /app/
+WORKDIR /app/
+
+# Install Python dependencies
+
+RUN pip3 install --upgrade pip -r requirements.txt
+RUN python3 ./manage.py makemigrations
+RUN python3 ./manage.py migrate
+RUN python3 ./banwebScrape.py
+RUN python3 ./manage.py custom_db_sync /tmp/writer.csv
+
+ARG DJANGO_SETTINGS_MODULE
+ENV DJANGO_SETTINGS_MODULE=$DJANGO_SETTINGS_MODULE
+
+ARG REACT_APP_ENDPOINT
+ENV REACT_APP_ENDPOINT=$REACT_APP_ENDPOINT
+
+ARG PROD_KEY
+ENV PROD_KEY=$PROD_KEY
+
+WORKDIR /app/webapp/
+
+RUN npm install --only=prod
+RUN npm update
+# Fix vulns
+RUN npm audit fix
+RUN npm run build
+RUN mkdir -p /app/static/
+RUN rsync -av --progress --exclude=static/ build/ /app/static/
+RUN rsync -av --progress build/static/* /app/static/
```

Comment:
Is there a reason we're using rsync here?

I think we could just do `cp -r build/* /app/static` and then `mv /app/static/static .` (and optionally `rm -r /app/static/static`).

Reply:
rsync was just the first tool I thought of with exclusion capabilities that weren't annoying to use.